autoPyTorch/pipeline/components/preprocessing/tabular_preprocessing/coalescer/MinorityCoalescer.py

```diff
@@ -0,0 +1,52 @@
+from typing import Any, Dict, Optional, Union
+
+from ConfigSpace.configuration_space import ConfigurationSpace
+from ConfigSpace.hyperparameters import (
+    UniformFloatHyperparameter,
+)
+
+import numpy as np
+
+from autoPyTorch.pipeline.components.preprocessing.tabular_preprocessing.coalescer.base_coalescer import BaseCoalescer
+from autoPyTorch.utils.common import HyperparameterSearchSpace, add_hyperparameter
+from autoPyTorch.utils.implementations import MinorityCoalescing
+
+
+class MinorityCoalescer(BaseCoalescer):
+    """
+    Groups together classes in a categorical feature if the frequency
+    of occurrence is less than minimum_fraction
+    """
+    def __init__(self, minimum_fraction: float, random_state: Optional[Union[np.random.RandomState, int]] = None):
+        super().__init__()
+        self.minimum_fraction = minimum_fraction
+        self.random_state = random_state
+
+    def fit(self, X: Dict[str, Any], y: Any = None) -> BaseCoalescer:
+
+        self.check_requirements(X, y)
+
+        self.preprocessor['categorical'] = MinorityCoalescing(minimum_fraction=self.minimum_fraction)
+        return self
+
+    @staticmethod
+    def get_properties(dataset_properties: Optional[Dict[str, Any]] = None) -> Dict[str, Union[str, bool]]:
+        return {
+            'shortname': 'MinorityCoalescer',
+            'name': 'Minority Feature-class coalescer',
+            'handles_sparse': False
```

Comment:
`handle_sparse` (imperative form)

Reply:
I don't think there is an agreement that the code is written in imperative form. Most of the core is written in the third person, as it was taken from auto-sklearn. You are very right that [google's code style](https://google.github.io/styleguide/pyguide.html) suggest that we use imperative and our docstrings are in google style format. I will recommend to disentangle this decision from this PR and have @nabenabe0928 and @ravinkohli  decide on this (and how to do so in automlcommon).

The reason for this is that `handles_*` is used not only on this new code but everywhere else.

Reply:
Ok, then let's go with `handles_*` now.
About the third person matter, have you ever seen any functions whose name is `gets_<something>` or `processes_<something>`?
But here, I got you and let's stay without the changes.